solidity/orders/script/Deploy.s.sol

```diff
@@ -19,24 +23,26 @@ contract Deploy is Script {
         scheduler = vm.envAddress("SCHEDULER_ADDRESS");
         factoryOwner = vm.envAddress("FACTORY_OWNER_ADDRESS");
         registryAddress = vm.envOr("REGISTRY_ADDRESS", address(0));
-        if (scheduler == address(0)) {
-            revert InvalidScheduler();
-        }
-        if (factoryOwner == address(0)) {
-            revert InvalidFactory();
-        }
+
+        if (scheduler == address(0)) revert InvalidScheduler();
+        if (factoryOwner == address(0)) revert InvalidFactory();
+
+        registrySalt = keccak256(abi.encodePacked(vm.envOr("REGISTRY_SALT", string("DEFAULT_REGISTRY_SALT"))));
```

Comment:
Can you please add this env into .env.example file?

Reply:
removed envs as they are not needed